tests/Avalonia.Controls.UnitTests/Primitives/SelectingItemsControlTests.cs

```diff
@@ -522,10 +522,19 @@ public void Removing_Selected_Item_Should_Clear_Selection()
             Assert.Equal(items[1], target.SelectedItem);
             Assert.Equal(1, target.SelectedIndex);
 
+            SelectionChangedEventArgs receivedArgs = null;
+
+            target.SelectionChanged += (_, args) => receivedArgs = null;
```

Comment:
This should read `receivedArgs = args`, right?

Reply:
Definitely, copy pasta gone wrong I guess. Tests still fail.